Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM registry.access.redhat.com/rhscl/ruby-23-rhel7
+
+# sshd
+USER root
+RUN yum install -y --setopt=tsflags=nodocs openssh-server libicu-devel && \
```

Comment:
Perhaps the Dockerfile shell syntax uses "/bin/sh" and that is not enough to enable SCL.
That's really really inconvenient :(

I'd suggest enabling the collection you need explicitly, something like:

```
scl enable xyz -- do something here ...
```